### ISSUE TITLE
update-build-version.sh: Allow setting git rev, instead of just NOGIT=1

### ISSUE THREAD

**Issue by flokli** [B-Lang-org/bsc#12]
My build environment doesn't provide neither the `git` executable nor the `.git` folder, so I can only set `NOGIT=1`, but miss all version information, as this falls back to a bunch of zeroes.

It'd be much more convenient if the version string could be passed in via make flags.

**Comment by quark17** [comment]
I agree that a flag is needed for passing in a version string to use.
